hack/jenkins/build.sh

```diff
@@ -62,7 +62,11 @@ git clean -fdx
 # docker images -q | xargs -r docker rmi
 
 # Build
-go run ./hack/e2e.go -v --build
+if [[ "${KUBE_FASTBUILD:-}" == "true" ]]; then
```

Comment:
well, @david-mcmahon is working on pulling out all of the release infra stuff, though I don't think it's 100% there yet.